src/mare/compiler/binary_verona.cr

```diff
@@ -37,18 +37,31 @@ class Mare::Compiler::BinaryVerona
       -fuse-ld=lld -rdynamic -static -fpic
       -lc -pthread -ldl -latomic
     }
+
+    ctx.link_libraries.each do |x|
+      link_args << "-l" + x
+    end
+
+    link_args <<
+      if ctx.options.release
+        "-O3"
+      else
+        "-O0"
+      end
+
     link_args << obj_filename
     link_args << VERONA_STATIC_LIB
-    link_args << "-o" << "main" # TODO: customizable output binary filename
+    link_args << "-o" << ctx.options.binary_name
 
     res = Process.run("/usr/bin/env", link_args, output: STDOUT, error: STDERR)
+    # res = Process.run("/usr/bin/env", ["strip", ctx.options.binary_name], output: STDOUT, error: STDERR)
```

Comment:
Looks like this line can be removed, right?
```suggestion
```

Reply:
Forgot to remove it

Reply:
I just was thinking if we need to strip our files

Reply:
We can actually strip files in release mode to reduce size of the executable

Reply:
I'm okay if you want to put the `strip` command in - I just want don't want to leave a commented-out line in. So I'd prefer to either remove it or uncomment it.